cpp/lib/include/Zeder.h

```diff
@@ -473,5 +473,10 @@ class SimpleZeder {
     inline const_iterator end() const { return entries_.end(); }
 };
 
```

Comment:
Please add one more blank line :-D

Reply:
Well, if it's our standard, that means that's also stringent for me, right ;)
Done, see #4137